core-metadata/src/main/java/org/apache/kylin/measure/topn/TopNCounter.java

```diff
@@ -43,8 +43,10 @@
 public class TopNCounter<T> implements Iterable<Counter<T>>, java.io.Serializable {
 
     public static final int EXTRA_SPACE_RATE = 50;
+    public static final int EXTRA_SORT_RATE = 10;
```

Comment:
It's better to add a property for this parameter, and set to a default value.

Reply:
done